.gitignore

```diff
@@ -1,4 +1,10 @@
 build/
 *.swp
-*.pdf
 
+# For Visual Studio Code files:
+.vscode/
```

Comment:
Please don't include stuff related to the personal development environment in project gitignores.

This belongs into your personal, user-wide gitignore, see:
https://git-scm.com/docs/gitignore (e.g. on linux, `$HOME/.config/git/ignore`

Reply:
Removed.